src/pages/billing/List.js

```diff
@@ -242,63 +236,59 @@ export const BillingList = ajaxCaller(class BillingList extends Component {
     }
   }
 
-  componentDidMount() {
-    this.refresh()
-  }
-
   render() {
-    const { billingProjects, isDataLoaded, filter, creatingBillingProject, creatingBillingAccount, updating } = this.state
+    const { billingProjects, isDataLoaded, creatingBillingProject } = this.state
+    const { queryParams: { selectedName } } = this.props
+    const breadcrumbs = `Billing > Billing Project`
     return h(Fragment, [
-      h(TopBar, { title: 'Billing' }, [
-        search({
-          wrapperProps: { style: { marginLeft: '2rem', flexGrow: 1, maxWidth: 500 } },
-          inputProps: {
-            placeholder: 'SEARCH BILLING PROJECTS',
-            onChange: e => this.setState({ filter: e.target.value }),
-            value: filter
+      h(TopBar, { title: 'Billing', href: Nav.getLink('billing') }, [
+        !!selectedName && div({
+          style: {
+            color: 'white', paddingLeft: '5rem', minWidth: 0, marginRight: '0.5rem'
           }
-        })
+        }, [
+          div(breadcrumbs),
+          div({
+            style: {
+              textOverflow: 'ellipsis', whiteSpace: 'nowrap',
+              fontSize: '1.25rem', overflowX: 'hidden'
+            }
+          }, [selectedName])
+        ])
       ]),
-      h(PageBox, [
-        div({ style: Style.cardList.toolbarContainer }, [
-          div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Billing Projects Management'])
-        ]),
-        div({ style: Style.cardList.cardContainer }, [
-          div([
-            h(NewBillingCard, {
-              newEntityLabel: 'New Project',
-              onClick: async () => {
-                try {
-                  await Auth.ensureBillingScope()
-                  this.setState({ creatingBillingProject: true })
-                } catch (error) {
-                  reportError('Failed to grant permissions', 'To create a new billing project, you must allow Terra to view your Google billing account(s).')
-                }
+      div({ style: { display: 'flex', flex: 1 } }, [
+        div({ style: { width: 330, boxShadow: '0 2px 5px 0 rgba(0,0,0,0.25)' } }, [
```

Comment:
The effective width of this seems to shrink and I get a horizontal scrollbar if my browser window is too narrow (and I don't think of it as terribly narrow).

Reply:
I'm not able to reproduce, could you include a screenshot?

Reply:
I can't reproduce now either. Must have been fixed by one of your recent changes.